dapps/digital-twin-detail/src/components/container/List.ts

```diff
@@ -45,6 +45,21 @@ export default class ContainerListComponent extends mixins(EvanComponent) {
 
   selectedValue = null;
 
+  /**
+   * Get the total number of list entries or `?` if we don't have access to the list.
+   */
+  get totalEntries(): number|string {
```

Comment:
Do both Tobi and you prefer `number|string` over `number | string`? I prefer the latter, but I would bow to the majority :)

Reply:
I didn't thought about, personally I like it without the spaces more, but everybody else seems to use spaces. So let's stick to the spaces.